pkg/github/github.go

```diff
@@ -275,14 +275,40 @@ func componentFromRepo(
 	// Check if there is a "releases/{suiteVersion}" branch
 	// If it exists, use that; if not, use master.
 	branch := "master"
-	hasReleaseBranch, err := checkForBranch(httpClient, "github_api", repo.Name, fmt.Sprintf("release/%s", suiteVersion))
+	hasReleaseBranch, err := checkForBranch(
```

Comment:
The logic for determine the branch name is getting a bit busy so I think it'd be good for us to move that into it's own method.

Reply:
thanks @doodlesbykumbi - I'll take that on as a follow on to this PR. expect to be tagged as the reviewer!!